src/js/components/ColumnCenter/ColumnCenter.js

```diff
@@ -0,0 +1,50 @@
+import React, { Children, cloneElement, forwardRef, useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+
+import { defaultProps } from '../../default-props';
+import { parseMetricToNum } from '../../utils';
+import { Grid } from '../Grid';
+import { Box } from '../Box';
+
+const ColumnCenter = forwardRef(
```

Comment:
This component name doesn't really make sense to me/I'm not sure I fully understand why it needs to be separate from the general "Columns" component. Is there any way to have a single component where the caller flags what kind of behavior they want? or based on the props they provide, Columns knows what to do?